---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

{% if site.author.googlescholar %}
  <div class="wordwrap">You can also find my articles on <a href="https://scholar.google.com/citations?user=4ajuBSkAAAAJ&hl=en">my Google Scholar profile</a>.</div>
{% endif %}

{% include base_path %}

{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %}

### &#x1F4C5;2025

*   Molecular dynamics-powered hierarchical geometric deep learning framework for protein-ligand interaction (**IF = 3.6, JCR Q1, IEEE Transactions on Computational Biology and Bioinformatics. (2025)**)  
    Mingquan Liu, Shuting Jin, Houtim Lai, Longyue Wang, **Jianmin Wang**, Xiangxiang Zeng\*  
   [IEEE Trans. Comput. Biol. Bioinform.](https://github.com/lmqfly/Dynamics-PLI) | [code](https://github.com/lmqfly/Dynamics-PLI)  

*   A Quantitative Ring Complexity Index for Profiling Ring Topology and Chemical Diversity (**Under Review**)  
    **Jianmin Wang**, Kai Xu, Tengfei Ma, Xudong Zhang, Pengsen Ma, Chunyan Li, Weiran Huang, Meng Wang,  Xiaojun Yao\*, Tao Song\*, Xiangxiang Zeng\*  
    [ChemRxiv. (2025)](https://doi.org/10.26434/chemrxiv-2025-mlqwl-v2) | [code](https://github.com/AspirinCode/QRCI)  

*   Self-supervised learning in drug discovery (**IF = 7.3, JCR Q1, SCIENCE CHINA Information Sciences. 2025**)  
    Yangyang Chen, Zixu Wang, **Jianmin Wang**, Yanyi Chu, Qingpeng Zhang, Zhong Alan Li, Xiangxiang Zeng\*  
    [Sci. China Inf. Sci. (2025)](https://link.springer.com/journal/11432)  

*   Instruction Multi-Constraint Molecular Generation Using a Teacher-Student Large Language Model (**IF = 4.4, JCR Q1, BMC Biology. 2025**)  
    Peng Zhou, **Jianmin Wang**, Chunyan Li, Zixu Wang, Yiping Liu, Siqi Sun, Jianxin Lin, Longyue Wang\*, Xiangxiang Zeng\*  
    [BMC Biol. (2025)](https://arxiv.org/abs/2403.13244) | [arXiv (2024)](https://arxiv.org/abs/2403.13244) | [code](https://github.com/HHW-zhou/TSMMG)  

*   Predicting mutation-disease associations through protein interactions via deep learning (**IF = 6.7, JCR Q1, IEEE Journal of Biomedical and Health Informatics. 2025**)  
    Xue Li, Ben Cao, **Jianmin Wang**, Xiangyu Meng, Shuang Wang, Yu Huang, Enrico Petretto, Tao Song\*  
    [J-BHI](http://dx.doi.org/10.1109/JBHI.2025.3541848) | [bioRxiv(2024)](https://doi.org/10.1101/2024.08.06.606730) | [code](https://github.com/xueleecs/CGM)  

*   Image-Based Generation for Molecule Design with SketchMol (**IF = 18.8, JCR Q1, Nature Machine Intelligence. (2025)**)  
    Zixu Wang, Yangyang Chen, Pengsen Ma, Zhou Yu, **Jianmin Wang**, Yuansheng Liu, Xiucai Ye, Tetsuya Sakurai, Xiangxiang Zeng\*  
   [Nat Mach Intell](https://doi.org/10.1038/s42256-025-00982-3) | [code](https://github.com/WangZiXubiubiu/SketchMol-v1)  

*   SurfDesign: Effective Protein Design on Molecular Surfaces (**Under review**)  
    Fang Wu, Shuting Jin, **Jianmin Wang**, Zerui Xu, xiangxiang Zeng, Jinbo Xu, Brian Hie  
   [Under review](https://openreview.net/forum?id=JCFJFBm5rE)  

### &#x1F4C5;2024

*   Interface-aware molecular generative framework for protein-protein interaction modulators (**IF = 7.1, JCR Q1, Journal of Cheminformatics (2024)**)  
    **Jianmin Wang**, Jiashun Mao, Chunyan Li, Hongxin Xiang, Xun Wang, Shuang Wang, Zixu Wang, Yangyang Chen, Yuquan Li, Kyoung Tai No\*, Tao Song\*, Xiangxiang Zeng\*  
    [J. Cheminform.](https://doi.org/10.1186/s13321-024-00930-0) | [bioRxiv 2023](https://doi.org/10.1101/2023.10.10.557742) | [code](https://github.com/AspirinCode/GENiPPI)  

*   Logical Rules Enhanced Multimodal Reasoning Based on Biomedical Knowledge Graph (**BIBM (2024)**)  
    Peifu Han, Tao Song\*, **Jianmin Wang**  
    [2024 IEEE International Conference on Bioinformatics and Biomedicine](https://ieeexplore.ieee.org/document/10821810)  

*   Activity cliff-informed contrastive learning for molecular property prediction (**Under Review, ChemRxiv 2024**)  
    Wanxiang Shen, Chao Cui, Xiaorui Su, Zaixi Zhang, Alejandro Velez-Arce, **Jianmin Wang**, Xiangcheng Shi, Yanbing Zhang, Jie Wu, Yuzong Chen\* and Marinka Zitnik\*  
    [ChemRxiv](https://doi.org/10.26434/chemrxiv-2023-5cz7s-v2) | [code](https://github.com/shenwanxiang/ACANet)  

*   Diffusion-based generative drug-like molecular editing with chemical natural language (**IF = 6.1, JCR Q1, Journal of Pharmaceutical Analysis (2024)**)  
    **Jianmin Wang**, Peng Zhou, Zixu Wang, Wei Long, Yangyang Chen, Kyoung Tai No, Dongsheng Ouyang\*，Jiashun Mao\* and Xiangxiang Zeng\*  
    [J. Pharm. Anal.](https://doi.org/10.1016/j.jpha.2024.101137) | [code](https://github.com/AspirinCode/DiffIUPAC)  

*   Improving compound-protein interaction prediction by focusing on intra-modality and inter-modality dynamics with a multimodal tensor fusion strategy (**IF = 4.4, JCR Q1, Computational and Structural Biotechnology Journal (2024)**)  
    Meng Wang, **Jianmin Wang**, Zhiwei Rong, Jianxin Ji, Chenjing Ma, Hesong Wang, Jia He, Yongzhen Song, Xuan Zhang, Yong Cao, Yanyan Dai, Menglei Hua, Ruihao Qin, Kang Li\* and Lei Cao\*  
    [Comput. Struct. Biotechnol. J.](https://doi.org/10.1016/j.csbj.2024.10.004) | [code](https://github.com/wangmeng-code/MMTF-CPI)  

*   An Image-enhanced Molecular Graph Representation Learning Framework  
    Hongxin Xiang, Shuting Jin, Jun Xia, Man Zhou, **Jianmin Wang**, Li Zeng, Xiangxiang Zeng\*  
    [IJCAI2024](https://doi.org/10.24963/ijcai.2024/675) | [code](https://github.com/HongxinXiang/IEM)  
    
*   Introducing enzymatic cleavage features and transfer learning realizes accurate peptide half-life prediction across species and organs (**IF = 6.8, JCR Q1, Briefings in Bioinformatics (2024)**)  
    Xiaorong Tan, Qianhui Liu, Yanpeng Fang, Sen Yang, Fei Chen, **Jianmin Wang** , Defang Ouyang, Jie Dong\*, Wenbin Zeng\*  
    [Brief. Bioinform.](https://doi.org/10.1093/bib/bbae350)
    
*   Exploring the conformational ensembles of protein-protein complex with transformer-based generative model (**IF = 5.5, JCR Q1, Journal of Chemical Theory and Computation (2024)**)  
    **Jianmin Wang**, Xun Wang, Yanyi Chu, Chunyan Li, Xue Li, Xiangyu Meng, Yitian Fang, Kyoung Tai No\*, Jiashun Mao\*, Xiangxiang Zeng\*  
   [J. Chem. Theory Comput.](https://doi.org/10.1021/acs.jctc.4c00255) | [bioRxiv](https://doi.org/10.1101/2024.02.24.581708) | [code](https://github.com/AspirinCode/AlphaPPImd)  
    
*   FraHMT: A fragment-oriented heterogeneous graph molecular generation model for target proteins (**IF = 5.6, JCR Q1, Journal of Chemical Information and Modeling (2024)**)  
    Shuang Wang, Dingming Liang, **Jianmin Wang**, Kaiyu Dong, Yunjing Zhang, Huicong Liang, Ximing Xu, Tao Song\*  
    [JCIM](https://doi.org/10.1021/acs.jcim.4c00252) | [code](https://github.com/llldddmmm/Code-FraHMT)  
    
*   DockingGA: Enhancing Targeted Molecule Generation using Transformer Neural Network and Genetic Algorithm with Docking Simulation (**IF = 4.0, JCR Q1, Briefings in functional genomics (2024)**)  
    Changnan Gao, Wenjie Bao, Shuang Wang, Jianyang Zheng, Lulu Wang, Yongqi Ren, Linfang Jiao, **Jianmin Wang\***, Xun Wang\*  
    [Brief. Funct. Genomics.](https://doi.org/10.1093/bfgp/elae011) | [code](https://github.com/Chinafor/DockingGA)  
    
*   A bidirectional interpretable compound-protein interaction prediction framework based on cross attention (**IF = 7.7, JCR Q1, Computers in Biology and Medicine (2024)**)  
    Meng Wang, **Jianmin Wang**, Zhiwei Rong, Liuying Wang, Zhenyi Xu, Liuchao Zhang, Jia He, Shuang Li, Lei Cao, Yan Hou, Kang Li\*  
    [COMPUT BIOL MED](https://doi.org/10.1016/j.compbiomed.2024.108239) | [code](https://github.com/wangmeng-code/CmhAttCPI)  
    
*   Local Scaffold Diversity-Contributed Generator for Discovering Potential NLRP3 Inhibitors (**IF = 5.6, JCR Q1, Journal of Chemical Information and Modeling (2024)**)  
    Weichen Bo, Yangqin Duan, Yurong Zou, Ziyan Ma, Tao Yang, Peng Wang,Tao Guo, Zhiyuan Fu, **Jianmin Wang**, Linchuan Fan, Jie liu, Taijin Wang\* and Lijuan Chen\*  
    [JCIM](https://doi.org/10.1021/acs.jcim.3c01818) | [code](https://github.com/wichen-2022/LSDC)  

### &#x1F4C5;2023
   
*   iupacGPT: IUPAC-based large-scale molecular pre-trained model for property prediction and molecule generation (**Under Review, ChemRxiv 2023**)  
    Jiashun Mao, **Jianmin Wang**, Kwang-Hwi Cho\* and Kyoung Tai No\*  
    [ChemRxiv](https://doi.org/10.26434/chemrxiv-2023-5kjvh) | [code](https://github.com/AspirinCode/iupacGPT)  
    
*   MOASL: Predicting drug mechanism of action through similarity learning with transcriptomic signature (**IF = 7.7, JCR Q1, Computers in Biology and Medicine (2023)**)  
   Likun Jiang, Susu Qu, Zhengqiu Yu, **Jianmin Wang**, Xiangrong Liu\*  
   [COMPUT BIOL MED](https://doi.org/10.1016/j.compbiomed.2023.107853) | [code](https://github.com/jianglikun/MOASL)  

*   A Multimodal Deep Learning Framework for Predicting PPI-Modulator Interactions (**IF = 5.6, JCR Q1, Journal of Chemical Information and Modeling (2023)**)  
    Heqi Sun, **Jianmin Wang\***, Hongyan Wu, Shenggeng Lin, Junwei Chen, Jinghua Wei, Shuai Lv, Yi Xiong\*, Dong-Qing Wei\*  
    [JCIM](https://doi.org/10.1021/acs.jcim.3c01527) | [code](https://github.com/sun-heqi/MultiPPIMI)  
    
*   Transformer-based molecular generative model for antiviral drug design (**IF = 5.6, JCR Q1, Journal of Chemical Information and Modeling (2023)**)  
    Jiashun Mao, **Jianmin Wang**, Amir Zeb, Kwang-Hwi Cho, Haiyan Jin, Jongwan Kim, Onju Lee, Yunyun Wang\* and Kyoung Tai No\*  
    [JCIM](https://doi.org/10.1021/acs.jcim.3c00536) | [code](https://github.com/AspirinCode/TransAntivirus)
    
*   Deep generative model for drug design from protein target sequence (**IF = 8.6, JCR Q1, Journal of Cheminformatics (2023)**)  
    Yangyang Chen, Zixu Wang, Lei Wang, **Jianmin Wang**, Pengyong Li, Dongsheng Cao, Xiangxiang Zeng, Xiucai Ye & Tetsuya Sakurai  
    [J Cheminform](https://doi.org/10.1186/s13321-023-00702-2) | [code](https://github.com/viko-3/TargetGAN)
    
*   Explore drug-like space with deep generative models (**IF = 4.2, JCR Q1, METHODS(2023)**)  
    **Jianmin Wang**, Jiashun Mao, Meng Wang, Xiangyang Le, Yunyun Wang\*  
    [Methods](https://doi.org/10.1016/j.ymeth.2023.01.004) | [code](https://github.com/AspirinCode/drug-likeness_space)  

### &#x1F4C5;2022    

*   Application of a deep generative model produces novel and diverse functional peptides against microbial resistance (**IF = 6.155, JCR Q1, Computational and Structural Biotechnology Journal (2022)**)  
    Jiashun Mao, Shenghui Guan, Yongqing Chen, Amir Zeb, Qingxiang Sun, Ranlan Lu, Jie Dong\*, **Jianmin Wang\***, Dongsheng Cao\*  
    [CSBJ](https://doi.org/10.1016/j.csbj.2022.12.029) | [code](https://github.com/AspirinCode/AMPTrans-lstm)
    
*   Accurate prediction of molecular properties and drug targets using a self-supervised image representation learning framework (**IF = 25.898, JCR Q1, Nature Machine Intelligence. (2022)**)  
    Xiangxiang Zeng, Hongxin Xiang, Linhui Yu, **Jianmin Wang**, Kenli Li, Ruth Nussinov, Feixiong Cheng  
   [Nat Mach Intell](https://www.nature.com/articles/s42256-022-00557-6) | [code](https://github.com/AspirinCode/ImageMol)  

*   Development of an innovative data-driven system to generate descriptive prediction equation of dielectric constant on small sample sets (**IF = 3.776, JCR Q2, Heliyon (2022)**)  
    Jiashun Mao, AmirZeb, Min-Sung Kim, Hyeon-Nae Jeon, **Jianmin Wang**, Shenghui Guan, Kyoung Tai NO  
    [Heliyon](https://doi.org/10.1016/j.heliyon.2022.e10011)  
    
*   Molormer: a lightweight self-attention-based method focused on spatial structure of molecular graph for drug–drug interactions prediction (**IF = 13.994, JCR Q1, Briefings in Bioinformatics (2022)**)  
    Xudong Zhang , Gan Wang, Xiangyu Meng, Shuang Wang, Ying Zhang , Alfonso Rodriguez-Paton, **Jianmin Wang\*** and Xun Wang\*  
    [Brief. Bioinform.](https://doi.org/10.1093/bib/bbac296) | [code](https://github.com/AspirinCode/Molormer)
    
*   De novo molecular design with deep molecular generative models for PPI inhibitors (**IF = 13.994, JCR Q1, Briefings in Bioinformatics (2022)**)  
    **Jianmin Wang**, Yanyi Chu , Jiashun Mao , Hyeon-Nae Jeon, Haiyan Jin, Amir Zeb, Yuil Jang, Kwang-Hwi Cho, Tao Song and Kyoung Tai No  
    [Brief. Bioinform.](https://doi.org/10.1093/bib/bbac285) | [code](https://github.com/AspirinCode/iPPIGAN)  
    
*   A transformer-based model to predict peptide–HLA class I binding and optimize mutated peptides for vaccine design (**IF = 25.898, JCR Q1, Nature Machine Intelligence (2022)**)  
    Yanyi Chu, Yan Zhang, Qiankun Wang, Lingfeng Zhang, Xuhong Wang, Yanjing Wang, Dennis Russell Salahub, Qin Xu, **Jianmin Wang**, Xue Jiang, Yi Xiong & Dong-Qing Wei  
    [Nat Mach Intell](https://doi.org/10.1038/s42256-022-00459-7) | [code](https://github.com/a96123155/TransPHLA-AOMP)
    
*   Geometry-Based Molecular Generation With Deep Constrained Variational Autoencoder (**IF = 14.255, JCR Q1, IEEE Transactions on Neural Networks and Learning Systems (2022)**)  
    Chunyan Li, Junfeng Yao, Wei Wei, Zhangming Niu, Xiangxiang Zeng, Jin Li, **Jianmin Wang**  
    [IEEE TNNLS](https://ieeexplore.ieee.org/abstract/document/9714718) | [code](https://github.com/anny0316/GEOM-CVAE)  

### &#x1F4C5;2021

*   A spatial-temporal gated attention module for molecular property prediction based on molecular geometry (**IF = 13.994, JCR Q1, Briefings in Bioinformatics (2021)**)  
    Chunyan Li, **Jianmin Wang**, Zhangming Niu, Junfeng Yao, Xiangxiang Zeng  
    [Brief. Bioinform.](https://doi.org/10.1093/bib/bbab078) | [code](https://github.com/anny0316/Drug3D-Net)  
    
*   MUFFIN: multi-scale feature fusion for drug–drug interaction prediction (**IF = 6.931, JCR Q1，Bioinformatics (2021)**)  
    Yujie Chen, Tengfei Ma, Xixi Yang, **Jianmin Wang**, Bosheng Song, Xiangxiang Zeng  
    [Bioinformatics](https://doi.org/10.1093/bioinformatics/btab169) | [code](https://github.com/xzenglab/MUFFIN)  
    
*   Design and linkage optimization of ursane-thalidomide-based PROTACs and identification of their targeted-degradation properties to MDM2 protein (**IF = 5.307, JCR Q1, Bioorganic Chemistry (2021)**)  
    Zhiwen Qi, Guliang Yang, Tao Deng, **Jianmin Wang**, Hao Zhou, Sergey A Popov, Elvira E Shults, Chengzhang Wang  
    [Bioorganic Chemistry](https://doi.org/10.1016/j.bioorg.2021.104901)  
    
*   Pretraining model for biological sequence data (**IF = 4.840, JCR Q1, Briefings in functional genomics (2021)**)  
    Bosheng Song, Zimeng Li, Xuan Lin, **Jianmin Wang**, Tian Wang, Xiangzheng Fu  
    [bifg](https://doi.org/10.1093/bfgp/elab025)  

### &#x1F4C5;2020

*   VGG16-T: A Novel Deep Convolutional Neural Network with Boosting to Identify Pathological Type of Lung Cancer in Early Stage by CT Images (**IF = 2.259, JCR Q3，International Journal of Computational Intelligence Systems (2020）**)  
    Shanchen Pang, Meng Fan, Xun Wang, **Jianmin Wang**, Tao Song, Xingguang Wang, Xiaochun Cheng  
    [ijcis](https://dx.doi.org/10.2991/ijcis.d.200608.001)  



